.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# application code
+/securedrop/ @redshiftzero
```

Comment:
not `securedrop/*`?

Reply:
That wouldn't cover things in `source_app` since the don't match the glob. So it has to be `/securedrop/`.